BricksAndHearts.UnitTests/ControllerTests/Property/PropertyControllerTests.cs

```diff
@@ -311,4 +312,49 @@ public void AddNewPropertyCancelPost_WithNoAddInProgress_RedirectsToViewProperti
         A.CallTo(() => propertyService.DeleteProperty(fakePropertyDbModel)).MustNotHaveHappened();
         result.Should().BeOfType<RedirectToActionResult>().Which.ActionName.Should().Be("ViewProperties");
     }
-}
+
+    [Fact]
+    public void ViewProperty_WithNonExistingProperty_RedirectToDashboard()

```

Comment:
See my comments on the `PropertyController`. I think if a property doesn't exist the user should see a 404 error page. then this test would be `ViewProperty_CalledOnNonExistentProperty_Returns404ErrorPage` or something like that

Reply:
ok sure done